### ISSUE TITLE
Docker.java:35-36: Continue implementing the rest of the...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#71]
The puzzle `3-571befa7` from #3 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/9abbafd0d62e336e7326c1ee82e1cef7ea1899e2/src/main/java/com/amihaiemil/docker/Docker.java#L35-L36

The puzzle was created by Mihai A on 28-Mar-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/G6LPQQV2P) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
The job [#71](https://github.com/amihaiemil/docker-java-api/issues/71) assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/871608862); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
Job `gh:amihaiemil/docker-java-api#71` already assigned to @amihaiemil, can't assign to @amihaiemil

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
The user @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) resigned from [#71](https://github.com/amihaiemil/docker-java-api/issues/71), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/remove_github_assignee.groovy
sun.reflect.NativeMethodAccessorImpl[-2] java.lang.reflect.InvocationTargetException: null
org.hamcrest.MatcherAssert[20] java.lang.AssertionError: HTTP response status is not equal to 200:
403 Forbidden [https://api.github.com/repos/amihaiemil/docker-java-api/issues/71]
Server: GitHub.com
Date: Mon, 09 Apr 2018 16:09:49 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 403 Forbidden
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4998
X-RateLimit-Reset: 1523293789
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
X-Runtime-Rack: 0.077777
X-GitHub-Request-Id: A27E:3171:184722A:30F0EE8:5ACB904D

{"message":"Must have admin rights to Repository.","documentation_url":"https://developer.github.com/v3/issues/#edit-an-issue"}
Expected: HTTP response with status 200
     but: was <403 Forbidden [https://api.github.com/repos/amihaiemil/docker-java-api/issues/71]
Server: GitHub.com
Date: Mon, 09 Apr 2018 16:09:49 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 403 Forbidden
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4998
X-RateLimit-Reset: 1523293789
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
X-Runtime-Rack: 0.077777
X-GitHub-Request-Id: A27E:3171:184722A:30F0EE8:5ACB904D

{"message":"Must have admin rights to Repository.","documentation_url":"https://developer.github.com/v3/issues/#edit-an-issue"}>
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
The job [#71](https://github.com/amihaiemil/docker-java-api/issues/71) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1071937467); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0pdd** [comment]
The puzzle `3-571befa7` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#86](https://github.com/amihaiemil/docker-java-api/issues/86) are still not solved.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#71](https://github.com/amihaiemil/docker-java-api/issues/71) is now out of scope

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#96](https://github.com/amihaiemil/docker-java-api/issues/96) are still not solved; solved: [#86](https://github.com/amihaiemil/docker-java-api/issues/86).

**Comment by 0pdd** [comment]
@0pdd 5 puzzles [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#96](https://github.com/amihaiemil/docker-java-api/issues/96), [#98](https://github.com/amihaiemil/docker-java-api/issues/98), [#99](https://github.com/amihaiemil/docker-java-api/issues/99) are still not solved; solved: [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#86](https://github.com/amihaiemil/docker-java-api/issues/86).

**Comment by 0pdd** [comment]
@0pdd 5 puzzles [#103](https://github.com/amihaiemil/docker-java-api/issues/103), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#98](https://github.com/amihaiemil/docker-java-api/issues/98), [#99](https://github.com/amihaiemil/docker-java-api/issues/99) are still not solved; solved: [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#86](https://github.com/amihaiemil/docker-java-api/issues/86), [#96](https://github.com/amihaiemil/docker-java-api/issues/96).
